src/shared/components/Login/Login.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React = require('react');
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import Login from './index';
+
+storiesOf('Components/Login', module).add(
+  'Login',
+  withInfo(`
+    The Login component allows to redirect to the Auth Provider page.
+
+    ~~~js
+      <Login loginURL="https://keycloak.com?redirect=https://nexus.com" />
```

Comment:
nexus.com domain already used by something else

Reply:
It's just an example but I'll use a something we own